src/api/customFetch.ts

```diff
@@ -0,0 +1,16 @@
+type FetchProps = RequestInit & {
+    body? : object
+}
+
+export const customFetch = async (url : string, options? : FetchProps) : Promise<Response> => {
+    const defaultHeaders = {
+        // 'X-Picsart-Plugin': 'Figma',
```

Comment:
i guess this one shall be un-commented?

Reply:
After uncommenting it is giving Cors error, I guess it is from the back end part which is not allowing any other header beside X-Picsart-API-Key

Reply:
well, we need to fix the cors, add back the header pls